server_test.go

```diff
@@ -653,7 +676,7 @@ func TestUNIXMetricsSSF(t *testing.T) {
 	path := filepath.Join(tdir, "test.sock")
 	config.SsfListenAddresses = []string{fmt.Sprintf("unix://%s", path)}
 	HTTPAddrPort++
-	f := newFixture(t, config)
+	f := newFixture(t, config, nil, nil)
```

Comment:
To repeat here what I said in chat before: This test depends (implicitly) on a metrics sink being created, but you pass a `nil` sink here. I would bet that if you create & pass that sink here (and also remove that `time.Sleep` stuff below & check that sink instead), the test will start passing.

Reply:
Was an unrelated problem!